app.js

```diff
@@ -297,6 +415,18 @@ app.post('/hubs/:hubSlug/start_activity', function(req, res){
   }
 })
 
+app.post('/hubs/:hubSlug/devices/:deviceSlug/activate_command', function(req, res){
```

Comment:
Did you have an opinion on the routes that I mentioned in #26?

I think we should go with:

`POST` `/hubs/:hub_slug/devices/:device_slug/commands/power-on`

This might make it even easier when using from other places, since it's all just the path, and there's no query params.

We should ship with this route, and in another pull we can migrate the activities route to be this format and deprecate the old one.